src/WebJobs.Extensions.DurableTask/DurabilityProvider.cs

```diff
@@ -598,6 +599,18 @@ public virtual bool TryGetTargetScaler(
         {
             targetScaler = null;
             return false;
+        }
+
+        /// <summary>
+        /// Streams the history of the specified orchestration instance as an enumerable of serialized history chunks.
+        /// </summary>
+        /// <param name="instanceId">The instance ID of the orchestration.</param>
+        /// <param name="jsonFormatter">The JSON formatter used to serialize the history chunks.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns>The enumerable of history chunks representing the orchestration's history.</returns>
+        public virtual Task<IEnumerable<string>> StreamOrchestrationHistoryAsync(string instanceId, JsonFormatter jsonFormatter, CancellationToken cancellationToken)
+        {
+            throw this.GetNotImplementedException(nameof(this.StreamOrchestrationHistoryAsync));
```

Comment:
I guess we would expect a PR at other backends repo? Can you link those in the PR description too when they are done?